doc/zmq.md

```diff
@@ -104,15 +104,14 @@ The topics are:
 
 Where the 8-byte uints correspond to the mempool sequence number.
 
-`rawtx`: Notifies about all transactions, both when they are added to mempool or when a new block arrives. This means a transaction could be published multiple times. First, when it enters the mempool and then again in each block that includes it. The messages are ZMQ multipart messages with three parts. The first part is the topic (`rawtx`), the second part is the serialized transaction, and the last part is a sequence number (representing the message count to detect lost messages).
```

Comment:
Does this functionality (tx events for every tx in a block) not work?

Reply:
It doesn't work. I've never seen that behavior. I changed from mempool long pooling to zmq interface on november 2020. Maybe it worked before, maybe not, and that's the expected behavior and it's wrong explained in the docs.

For me, the actual behaviour it's fine, and the one explained in the docs it's trickier, since it forces you into checking if an incoming transaction is from last block or was added to mempool to be mined.

Reply:
Which Bitcoin Core version (e.g. v22.0 or a `master` build) are you running?

Last time I checked this functionality (mempool + block tx via `rawtx`) did work. IIRC the `rawtx` topic gave me both mempool and block tx around taproot activation. I remember adding a custom ZMQ publisher for my taproot activation monitoring livestream. I also PR'ed a WIP implementation of a `rawmempooltx` publisher in https://github.com/bitcoin/bitcoin/pull/23624 as other users were getting mempool and block transactions through `rawtx` too, but were only looking for mempool transactions.

Reply:
Currently running v22.0. But I've used `hashtx`. Different behaviors for each one? I'm going to do test for both and comment here the results ASAP.